app/seeders/demo_data/project_seeder.rb

```diff
@@ -156,11 +160,15 @@ def seed_versions(project, key)
     def seed_board(project)
       Forum.create!(
         project: project,
-        name: translate_with_base_url('seeders.demo_data.board.name'),
-        description: translate_with_base_url('seeders.demo_data.board.description')
+        name: translate_with_base_url("seeders.#{OpenProject::Configuration['edition']}.demo_data.board.name"),
+        description: translate_with_base_url("seeders.#{OpenProject::Configuration['edition']}.demo_data.board.description")
       )
     end
 
+    def project_key(key)
+      "seeders.#{OpenProject::Configuration['edition']}.demo_data.projects.#{key}"
```

Comment:
This line is written very often in this changes. Please consider making it a public method in the super class or at least use the methods you defined.